docs/admin/admission-controllers.md

```diff
@@ -142,6 +143,15 @@ applies a 0.1 CPU requirement to all Pods in the `default` namespace.
 
 See the [limitRange design doc](../design/admission_control_limit_range.md) and the [example of Limit Range](limitrange/) for more details.
 
+### InitialResources (experimental)
+
+This plug-in observes the incoming request to create a pod and for each container without Resources specified (neither Request nor Limit)
+set Request in a smart way. The prediction is made based on historical usage of containers running the same image.
+If there is not enough data to make a decission the Request is left unchanged.
+When InitialResources set Request it will also add apriopriate annotation to a pod to let the user know about it.
```

Comment:
When the plug-in sets a compute resource request, it annotates the pod with information on what compute resources it auto-populated.